### ISSUE TITLE
reload command does not also reload nginx configuration

### ISSUE THREAD

**Issue by Dibble** [Mashape/kong#1484]
### Summary

According to #114 it seems that running `kong reload` should also cause nginx to reload its configuration, however nginx does not reload with the new config.

### Steps To Reproduce

1. `kong start`
2. modify the nginx configuration in kong.yml (eg. add or edit a server block)
3. `kong reload`
4. nginx will still be using the old configuration

Example configuration change:
```
server {
  listen 8888;
  location / {
    return 200 'Hello World!;
  }
}
```

### Additional Details & Logs

I have seen this using the code from both `master` and `next` branches, however the behaviour appears to be slightly different.

On `master` it appears that the configuration changes are not being copied into the nginx working directory, so adding a call to `nginx:prepare()` before the reload seems to fix it. The reload works with that change, but I haven't thoroughly tested it for side-effects or other bugs.

Using the `next` branch the configuration is copied to the nginx working directory, but the reload doesn't cause nginx to update. I haven't yet figured out why.

**Comment by thibaultcha** [comment]
Thanks for the report! I just proposed #1485 to resolve this.

**Comment by Dibble** [comment]
Excellent!

It worked in the quick test I did on my dev machine.

**Comment by thibaultcha** [comment]
Considering this resolved in the upcoming 0.9 release, thanks!
